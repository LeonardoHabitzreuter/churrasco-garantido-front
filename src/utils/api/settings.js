export const BASE_API_URL = process.env.apiHost || 'http://localhost:3001'
