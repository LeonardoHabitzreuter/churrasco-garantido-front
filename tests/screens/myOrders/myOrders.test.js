import React from 'react'
import { shallow } from 'enzyme'
import MyOrders from '../../../src/screens/myOrders'
import api from '../../../src/utils/api'
api.getUser = jest.fn()

describe('My orders component', () => {
  test('should renders properly', () => {
    const wrapper = shallow(<MyOrders location={{}} />)
    expect(wrapper).toMatchSnapshot()
  })
})
