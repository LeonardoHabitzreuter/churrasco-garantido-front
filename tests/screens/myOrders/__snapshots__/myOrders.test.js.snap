// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`My orders component should renders properly 1`] = `
<div
  className="col-sm-8"
  style={
    Object {
      "display": "inline-grid",
    }
  }
>
  <PageHeader
    bsClass="page-header"
  >
    Meus pedidos
  </PageHeader>
  <Loader
    loading={true}
    size="large"
  />
  <Alert
    handleDismiss={[Function]}
    messages={Array []}
    show={false}
    style=""
  />
  <div>
    <div
      className="col-sm-4"
    >
      <h3>
        <Label
          bsClass="label"
          bsStyle="default"
        >
          CNPJ
        </Label>
      </h3>
      <input
        min="1"
        onChange={[Function]}
        type="text"
        value=""
      />
    </div>
    <div
      className="col-sm-4"
    >
      <h3>
        <Label
          bsClass="label"
          bsStyle="default"
        >
          Pedido
        </Label>
      </h3>
      <input
        min="1"
        onChange={[Function]}
        type="text"
        value=""
      />
    </div>
    <div
      className="col-sm-4"
    >
      <h3>
        <Label
          bsClass="label"
          bsStyle="default"
        >
          Açoes
        </Label>
      </h3>
      <Button
        bsStyle="primary"
        onClick={[Function]}
        type="button"
      >
        Filtrar
      </Button>
    </div>
  </div>
  <div
    style={
      Object {
        "paddingTop": "5rem",
      }
    }
  >
    <Table
      columns={
        Object {
          "actions": "Ações",
          "code": "Código do pedido",
          "products": "Itens",
        }
      }
      lines={Array []}
    />
  </div>
</div>
`;
